### PR TITLE
changes to contract view and edit mode

### PR DESCRIPTION
**Pull request by Eelsie**

### CODE REVIEW

File: src/leases/components/leaseSections/contract/Contract.js [hunk 1]
```diff
@@ -27,40 +27,75 @@ const Contract = ({contract}: Props) => {
...
               <label>Allekirjoituspäivämäärä</label>
               <p>{contract.signing_date ? helpers.formatDate(get(contract, 'signing_date', '–')) : '–'}</p>
-
-              <label>Vuokravakuusnumero</label>
-              <p>{contract.lease_deposit_number ? get(contract, 'lease_deposit_number', '–') : '–'}</p>
-
+            </Column>
+            <Column medium={4}>
+              <label>Kommentti allekirjoitukselle</label>
+              <p>{contract.signing_date_comment ? get(contract, 'signing_date_comment', '–') : '–'}</p>
```
Comment (jorilindell) [inline src/leases/components/leaseSections/contract/Contract.js]: Default string (-) is defined twice. You get same end result with
<p>{get(contract, 'signing_date_comment', '–')}</p>

Same thing on rows 43, 59, 73, 87, 91, 95

File: src/leases/components/leaseSections/contract/Contract.js [hunk 2]
```diff
@@ -27,40 +27,75 @@ const Contract = ({contract}: Props) => {
...
+            </Column>
+          </Row>
+          {contract.pledge_books.length > 0 &&
+          <Row>
+            <Column>
+              <p className='green-subtitle'>Panttikirjat</p>
+            </Column>
+          </Row>
+          }
+          {contract.pledge_books.length > 0 && contract.pledge_books.map((pledge_book, index) =>
```
Comment (jorilindell) [inline src/leases/components/leaseSections/contract/Contract.js]: Page will crash if contract.pledge_books is null. I would be good to check is contract.pledge_books defined
e.g.
{contract.pledge_books && contract.pledge_books.length > 0 && contract.pledge_books.map((pledge_book, index) =>

File: src/leases/components/leaseSections/contract/Contract.js (hunk 1 above)
Comment (Eelsie) [reply src/leases/components/leaseSections/contract/Contract.js]: The purpose is to show '–' also if null

File: src/leases/components/leaseSections/contract/Contract.js (hunk 2 above)
Comment (Eelsie) [reply src/leases/components/leaseSections/contract/Contract.js]: of course! forgot that :/

**Pull request merged**